build.gradle

```diff
@@ -104,7 +104,9 @@ dependencies {
     // =============
     testImplementation 'org.testcontainers:postgresql:1.12.5'
     if (getGradle().getStartParameter().taskNames.contains('runServer')) {
-        runtimeClasspath 'org.testcontainers:postgresql:1.12.5'
+        runtimeClasspath 'org.springframework.cloud:spring-cloud-starter:2.2.3.RELEASE'
+        runtimeClasspath "com.playtika.testcontainers:embedded-postgresql:1.78"
```

Comment:
This dependency is linked to from the official Test Containers GitHub as being the appropriate choice for Spring Boot applications. It gives us benefits like bootstrap properties for configuration rather than query parameters on the JDBC URL.